### PR TITLE
Adding my name to list

### LINKED ISSUE #5: Onboard Saurabh
- [ ] Schedule 2 1 hour meetings their first or first and second week
- [x] Make sure they have git installed
- [ ] Git intro 1
- [ ] Complete sections 1 & 2 of https://learngitbranching.js.org/
- [x] Make sure they have GitHub setup and the correct access
- [ ] Git intro 2

### PR DESCRIPTION
**Pull request by mogres**
Problem
=======
What is the problem this work solves, including
Closes #5 

Solution
========
What I/we did to solve this problem
- Added my name to the list
with @meganrm 

## Type of change
Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: index.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #adding two names to the new employee array
-employees = ['Nivedita', 'Na Hyung']
+employees = ['Nivedita', 'Na Hyung', 'Saurabh']
```
Comment (prismofeverything) [inline index.py:2]: Looks great!

**Review by prismofeverything — APPROVED** [review]

File: index.py (hunk 1 above)
Comment (meganrm) [inline index.py:2]: Can you update the comment, it still says "two names" but we have three now?

**Review by meganrm — CHANGES_REQUESTED** [review]
Please update the comment.

File: index.py (hunk 1 above)
Comment (mogres) [reply index.py:2]: - [ ] update comment

**Review by meganrm — APPROVED** [review]

**Pull request merged**